awscli/examples/gamelift/create-fleet.rst

```diff
@@ -1,11 +1,11 @@
 **Example 1: To create a basic Linux fleet**
 
-The following ``create-fleet`` example creates a minimally configured fleet of on-demand Linux instances to host a custom server build. You can complete the configuration by using ``update-fleet``. ::
+The following ``create-fleet`` example creates a minimally configured fleet of on-demand Linux instances to host a custom server build. You can complete the configuration by using ``UpdateFleet()``. ::
```

Comment:
From the perspective of the CLI, shouldn't the `update-fleet` command be used?

Reply:
These are now fixed. You were right. :-)